src/studio/src/designer/backend/Controllers/TextController.cs

```diff
@@ -97,27 +98,29 @@ public IActionResult GetLanguages(string org, string app)
         [HttpPost]
         public IActionResult SaveResource([FromBody] dynamic jsonData, string id, string org, string app)
         {
-            id = id.Split('-')[0];
+            // id = id.Split('-')[0];
             JObject json = jsonData;
 
-            JArray resources = json["resources"] as JArray;
-            string[] duplicateKeys = resources.GroupBy(obj => obj["id"]).Where(grp => grp.Count() > 1).Select(grp => grp.Key.ToString()).ToArray();
-            if (duplicateKeys.Length > 0)
-            {
-                return BadRequest($"Text keys must be unique. Please review keys: {string.Join(", ", duplicateKeys)}");
-            }
+            Console.WriteLine(org + app + id);
 
-            JArray sorted = new JArray(resources.OrderBy(obj => obj["id"]));
-            json["resources"].Replace(sorted);
+            // JArray resources = json["resources"] as JArray;
```

Comment:
Commenting out this code will possibly give a false positive when it comes to passing the tests.
If none of these changes are intentionally going to be distributed you must revert the files back to their original.